apikana-maven-plugin/src/main/java/org/swisspush/apikana/AbstractApikanaMojo.java

```diff
@@ -75,9 +77,57 @@ public abstract class AbstractApikanaMojo extends AbstractMojo {
      *
      * <p>Default: 0.0.0</p>
      */
-    @Parameter(defaultValue = "0.0.0", property = "apikana.defaults.version")
+    @Parameter(defaultValue = "0.0.80", property = "apikana.defaults.version")
     protected String apikanaDefaultsVersion;
 
+    /**
+     * The java package that should be used.
+     */
+    @Parameter(property = "apikana.java-package")
+    protected String javaPackage;
+
+    /**
+     * The main API file (yaml or json).
+     */
+    @Parameter(defaultValue = "src/openapi/api.yaml", property = "apikana.api")
+    protected String api;
+
+    /**
+     * The typescript version which should be added to package.json
+     */
+    @Parameter(property = "apikana.typescript.version")
+    protected String typescriptVersion;
+
+    /**
+     * The scope used to name the package
+     */
+    @Parameter(property = "apikana.scope")
+    protected String scope;
+
+    /**
+     * The apikana API type
+     */
+    @Parameter(defaultValue = "rest-api", property = "apikana.type")
+    protected String type;
```

Comment:
I think we should describe more what this is. It looks like some kind of enum. But what values are actually possible?

Reply:
Added the possible values.